elmslie-android/src/main/java/vivid/money/elmslie/android/screen/ElmDelegate.kt

```diff
@@ -10,6 +10,7 @@ interface ElmDelegate<Event : Any, Effect : Any, State : Any> {
 
     val initEvent: Event
     val storeHolder: StoreHolder<Event, Effect, State>
+    val shouldStopOnProcessDeath: Boolean
```

Comment:
It brokes API a little bit because users should implement this for their own Fragments such as BottomSheetFragments, but with the help of Fragment.parentStopElmOnProcessDeath() it will be easy

Reply:
It would be better to add a javadoc for this, since it's not a very obvious feature

Reply:
I think since this property is moved to the ElmDelefate interface the StopElmOnProcessDeath interface is no loger needed. All delegates could override it insted of impleneting the interface

Reply:
If we use traverse by fragment hierarchy interface will be necessary (mark parents fragment by this interface and understand that child fragment should be stopped). Let's finish the discussion in the thread about traversing and return to this question.